### PR TITLE
Predict and exploit Blech House and Protestors guaranteed noncombats

### PR DESCRIPTION
**Pull request by jaspercb**

### CODE REVIEW

**Comment by jaspercb** [comment]
it's possible you don't want my list of day 1 softcore pulls cluttering this up for other people, but I genuinely think turngen is the most important thing on day 1 in this path, so *shrug*

File: RELEASE/scripts/sl_ascend.ash [hunk 1]
```diff
@@ -12573,34 +12627,65 @@ boolean L11_redZeppelin()
 		return ccAdv($location[A Mob Of Zeppelin Protesters]);
 	}
 
-	if(item_amount($item[lynyrd snare]) > 0 && get_property("_lynyrdSnareUses").to_int() < 3 && my_hp() > 150)
-	{
-		return ccAdvBypass("inv_use.php?pwd=&whichitem=7204&checked=1", $location[Noob Cave]);
-	}
+	// TODO: create lynyrd skin items
```
Comment (MatthewDarling) [inline RELEASE/scripts/sl_ascend.ash]: I tried to do this recently, but couldn't without [Armorcraftiness](http://kol.coldfront.net/thekolwiki/index.php?title=Armorcraftiness). Is there another way to create the Lynyrd Skin equipment in Dark Gyfte?

Comment (jaspercb) [reply RELEASE/scripts/sl_ascend.ash]: That's right, it's not possible in DG but is in other paths.

File: RELEASE/scripts/sl_ascend.ash [hunk 2]
```diff
@@ -1603,6 +1603,45 @@ int handlePulls(int day)
...
+
+			for(int i=0; i<available_organspace-available_bloodbags; i++)
+			{
+				pullXWhenHaveY($item[vampagne], 1, i);
+			}
+
+			// this is a somewhat meat-heavy path, especially if
+			// buying torso awaregness and desert bus pass
+			pullXWhenHaveY($item[gold wedding ring], 1, 0);
+			sl_autosell(1, $item[gold wedding ring]);
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend.ash]: I've been getting by just fine on meat in hardcore, this pull doesn't really seem necessary to me.

Comment (jaspercb) [reply RELEASE/scripts/sl_ascend.ash]: Fair! I'll remove it.

**Pull request merged**